doc/docker_integration.md

```diff
@@ -1,6 +1,10 @@
 Docker integration
 ===============================================================================
 
+**Note:** This page is mainly about building Haskell packages inside docker containers.
+If you want to deploy your built Haskell programs into a docker container,
+look [here](../GUIDE.md) instead.
```

Comment:
I think this should just be `GUIDE.md`, the file is in the same directory. And what about providing an anchor to the appropriate part of the page?